flake-info/src/data/import.rs

```diff
@@ -27,7 +26,7 @@ pub enum FlakeEntry {
         attribute_name: String,
         name: String,
         version: String,
-        platforms: Vec<System>,
+        platforms: Vec<String>,
         outputs: Vec<String>,
         default_output: String,
         description: Option<String>,
```

Comment:
I'd like to see types for the specific attributes (even if its just type aliases to String), but that's for another time